### PR TITLE
Update readme, add .ico for applet

### PR DESCRIPTION
**Pull request by ghost**
Updated the readme to be a tad more fancier/descriptive, and added an icon for the control application.

NOTE: The overlay.py script was updated in this case with the icon bitmap, not sure if that is what 100% runs the main application. Check line 42.

### CODE REVIEW

File: overlay.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 root.attributes('-topmost', True)
 root.geometry("1920x1080+0+0")
 root.overrideredirect(True)
+root.iconbitmap("FormicRadarIcon.ico")
```
Comment (waarongoo) [inline overlay.py]: Remove this as it's unnecessary thx

File: README.md [hunk 2]
```diff
@@ -1,20 +1,39 @@
-# FormicRadar Revive
-# Download from the releases tab
+# FormicRadar Revive 📡
```
Comment (waarongoo) [inline README.md]: Change relevant to "FormicAcid Radar Revive"

File: README.md [hunk 3]
```diff
@@ -1,20 +1,39 @@
...
 
+![Project Banner](readme_banner.png?raw=true)
 
+![Static Badge](https://img.shields.io/badge/license-MIT-red?link=https%3A%2F%2Fopensource.org%2Flicense%2Fmit) ![Static Badge](https://img.shields.io/badge/mom_made-pizza_rolls-blue?link=https%3A%2F%2Fopensource.org%2Flicense%2Fmit)
```
Comment (waarongoo) [inline README.md:5]: Lol

File: README.md [hunk 4]
```diff
@@ -1,20 +1,39 @@
-# FormicRadar Revive
-# Download from the releases tab
+# FormicRadar Revive 📡
 
+![Project Banner](readme_banner.png?raw=true)
```
Comment (waarongoo) [inline README.md:3]: Good work here! Looks sick

File: README.md [hunk 5]
```diff
@@ -1,20 +1,39 @@
...
 
-## License
+> 🌍 **Best used for ATC24** or Anything PTFS ATC!
 
-This project is licensed under the [MIT License](LICENSE).
+
+## Installation 🔧
+
+1. Get the latest [release](<https://github.com/awdev1/FormicAcid-Radar-Revive/releases>)
+2. Unzip the folder
```
Comment (waarongoo) [inline README.md]: Un zip folder to a suitable folder ig

**Review by waarongoo — CHANGES_REQUESTED** [review]
Thanks for your work! Please review these comments before I merge this PR

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: Most likely not needed as the way windows unzips files

Comment (waarongoo) [reply README.md:23]: Macs:

Comment (awdev1) [reply README.md]: Some software just extract files directly+ just some added protection to people who don't know how zips work

**Review by awdev1 — APPROVED** [review]

**Pull request merged**